tests/unit/custom_metric_test.py

```diff
@@ -0,0 +1,208 @@
+import numpy as np
+
+from src.tum_dlr_automl_for_eo.utils import custom_metrics
+
+random_acc_list = np.array([
+    [17, 74, 77, 81, 61, 84, 10, 82, 56, 66, 44, 40, 72, 23, 6, 59,
+     98, 34, 26, 5],
+    [15, 67, 3, 47, 8, 82, 36, 81, 29, 37, 37, 95, 67, 6, 44, 4,
+     79, 27, 12, 60],
+    [30, 57, 47, 29, 23, 92, 93, 90, 55, 95, 59, 38, 1, 93, 55, 97,
```

Comment:
Actually, test cases doesn't need to be so long. They are just for demonstration, is it related with positive and negative resistance?

Reply:
We need to track whole all models at each epoch to see best(worst) 25%. So, using less models would not make sense to test the functions.